geniza/corpus/models.py

```diff
@@ -525,6 +525,22 @@ def editions(self):
             "content", "source"
         )
 
+    def digital_editions(self):
+        """All footnotes for this document where the document relation includes
+        edition AND the footnote has content."""
+        return (
+            self.footnotes.filter(doc_relation__contains=Footnote.EDITION)
+            .filter(content__isnull=False)
+            .order_by("content", "source")
+        )
+
+    def editors(self):
+        """All unique authors of digital editions for this document."""
+        editor_pks = []
+        for de in self.digital_editions():
+            editor_pks += [author.pk for author in de.source.authors.all()]
+        return Creator.objects.filter(pk__in=editor_pks)
```

Comment:
Should be possible to get this into a single query, how about this:
```suggestion
        return Creator.objects.filter(source__footnote__doc_relation__contains=Footnote.EDITION, content_isnull=False, source__footnote__document=self).distinct()
```